### PR TITLE
blockdev_mirror_to_rbd:remove image1 ceph set limiation

### PR DESCRIPTION
**Pull request by aliang123**
Remove ceph set limitation on system disk to make it possible to start system disk under ceph.

Signed-off-by: Aihua Liang <aliang@redhat.com>
id:2150701

### CODE REVIEW

**Comment by aliang123** [comment]
(1/1) Host_RHEL.m8.u8.product_rhel.ceph.raw.virtio_scsi.up.virtio_net.Guest.RHEL.8.8.0.x86_64.io-github-autotest-qemu.blockdev_mirror_to_rbd.q35: PASS (165.02 s)

**Comment by aliang123** [comment]
@zhencliu @XueqiangWei , please help to review, thanks.

**Comment by XueqiangWei** [comment]
@aliang123 Please check the spelling in commit message, it should be "limitation" instead of "limiation"?

File: qemu/tests/cfg/blockdev_mirror_to_rbd.cfg [hunk 1]
```diff
@@ -29,7 +29,6 @@
     storage_type_local = directory
     enable_ceph_mirror1 = yes
     enable_ceph_data1 = no
-    enable_ceph_image1 = no
```
Comment (XueqiangWei) [inline qemu/tests/cfg/blockdev_mirror_to_rbd.cfg:32]: According to the comment in line 28, it's for local image. Why don't we add "enable_ceph_image1 = yes" under "ceph:"? like data1 in line 31 and line 33. If I was wrong, please correct me. Thanks.

Comment (aliang123) [reply qemu/tests/cfg/blockdev_mirror_to_rbd.cfg:32]: > hi @XueqiangWei , this case is for: local data image -----(mirror to)----> ceph image in order to configure the ceph backend, we have to use the ceph storage backend(only ceph), in this case, enable_ceph is 'yes' by default, so for all local images, we have to disable it with enable_ceph_xxx = no
> 
> hi @aliang123 , IMO, the purpose of this code update is that you'd like to add this case into your ceph test loop, right?

Yes, @zhencliu, when run tests with backend ceph, the system image should be on ceph by default. In our original case design, we just limit system storage to local storage, that's not reasonable, so correct it here.

**Review by zhencliu — APPROVED** [review]

File: qemu/tests/cfg/blockdev_mirror_to_rbd.cfg (hunk 1 above)
Comment (XueqiangWei) [reply qemu/tests/cfg/blockdev_mirror_to_rbd.cfg:32]: Thank you Zhenchao, Aihua for your explanation.
LGTM

**Pull request merged**